package.json

```diff
@@ -5,8 +5,8 @@
   "private": true,
   "repository": "github.com/kbase/narrative",
   "devDependencies": {
-    "bower": "1.8.4",
-    "geckodriver": "1.6.1",
+    "bower": "^1.8.4",
```

Comment:
IMO It is dangerous to use non-deterministic dependencies specs like ^SEMVER in a top level app like this -- it means that the build will change over time even though no-one made changes to the codebase. (Libraries must use them, because of the way JS dependency managers work.) Even though this usage of ^SEMVER _should_ ensure that the build doesn't break (because it will only allow changes in the second and third server positions, which are always supposed to be backwards compatible), in the JS world "safe" dependency changes can sometimes break. 

I do think it is good to update the deps from time to time not just to to pick up improvements but more importantly to reduce the amount of upstream code change when one does update deps. In the best of worlds, one would inspect the diff in the upstream dependency to see what exactly has changed to evaluate whether it is (a) worth the dependency update and (b) to understand what is different and whether it might affect something. In reality, that is quite difficult to do!